apps/anoma_node/lib/node/utility/indexer.ex

```diff
@@ -76,10 +76,33 @@ defmodule Anoma.Node.Utility.Indexer do
     GenServer.call(name, flag)
   end
 
+  @doc """
+  I return a list of all unspent resources.
+  """
+  @spec unspent_resources(String.t()) :: [Resource.t()]
+  def unspent_resources(node_id) do
+    name = Registry.via(node_id, __MODULE__)
+    GenServer.call(name, :unspent_resources)
+  end
+
   ############################################################
   #                    Genserver Behavior                    #
   ############################################################
 
+  def handle_call(:unspent_resources, _from, state) do
```

Comment:
Not sure what is happening here. We already have the `:resources` flag that does almost exactly this without needing to convert back to an elixir resource struct